runtime/compiler/aarch64/codegen/J9AheadOfTimeCompile.cpp

```diff
@@ -36,8 +40,1040 @@ void J9::ARM64::AheadOfTimeCompile::processRelocations()
+         binaryTemplate->_symbolType = symbolType;
+
+         cursor += sizeof(TR_RelocationRecordSymbolFromManagerBinaryTemplate);
+         }
+         break;
+
+      case TR_ValidateClass:
+         {
+         *(uintptrj_t*)cursor = (uintptrj_t)relocation->getTargetAddress(); // Inlined site index
+         cursor += SIZEPOINTER;
+
+         TR::AOTClassInfo *aotCI = (TR::AOTClassInfo*)relocation->getTargetAddress2();
+         *(uintptrj_t*)cursor = (uintptrj_t) aotCI->_constantPool;
+         cursor += SIZEPOINTER;
+
+         *(uintptrj_t*)cursor = (uintptrj_t) aotCI->_cpIndex;
+         cursor += SIZEPOINTER;
+
+         uintptr_t classChainOffsetInSharedCache = sharedCache->offsetInSharedCacheFromPointer(aotCI->_classChain);
+         *(uintptrj_t *)cursor = classChainOffsetInSharedCache;
+         cursor += SIZEPOINTER;
+         }
+         break;
+
+      case TR_ValidateStaticField:
+         {
+         *(uintptrj_t*)cursor = (uintptrj_t)relocation->getTargetAddress(); // Inlined site index
+         cursor += SIZEPOINTER;
+
+         TR::AOTClassInfo *aotCI = (TR::AOTClassInfo*)relocation->getTargetAddress2();
+         *(uintptrj_t*)cursor = (uintptrj_t) aotCI->_constantPool;
+         cursor += SIZEPOINTER;
+
+         *(uintptrj_t*)cursor = (uintptrj_t) aotCI->_cpIndex;
+         cursor += SIZEPOINTER;
+
+         void *romClass = (void *)fej9->getPersistentClassPointerFromClassPointer(aotCI->_clazz);
+         uintptr_t romClassOffsetInSharedCache = sharedCache->offsetInSharedCacheFromPointer(romClass);
+         *(uintptrj_t *)cursor = romClassOffsetInSharedCache;
+         cursor += SIZEPOINTER;
+         }
+         break;
+
+      case TR_ValidateArbitraryClass:
+         {
+         TR::AOTClassInfo *aotCI = (TR::AOTClassInfo*) relocation->getTargetAddress2();
+         TR_OpaqueClassBlock *classToValidate = aotCI->_clazz;
+
+         //store the classchain's offset for the classloader for the class
+         uintptrj_t classChainOffsetInSharedCacheForCL = sharedCache->getClassChainOffsetOfIdentifyingLoaderForClazzInSharedCache(classToValidate);
+         *(uintptrj_t *)cursor = classChainOffsetInSharedCacheForCL;
+         cursor += SIZEPOINTER;
+
+         //store the classchain's offset for the class that needs to be validated in the second run
+         void *romClass = (void *)fej9->getPersistentClassPointerFromClassPointer(classToValidate);
+         uintptrj_t *classChainForClassToValidate = (uintptrj_t *) aotCI->_classChain;
+         uintptr_t classChainOffsetInSharedCache = sharedCache->offsetInSharedCacheFromPointer(classChainForClassToValidate);
+         *(uintptrj_t *)cursor = classChainOffsetInSharedCache;
+         cursor += SIZEPOINTER;
+         }
+         break;
+
+      case TR_HCR:
+         {
+         flags = 0;
+         if (((TR_HCRAssumptionFlags)((uintptrj_t)(relocation->getTargetAddress2()))) == needsFullSizeRuntimeAssumption)
+        	   flags = needsFullSizeRuntimeAssumption;
```

Comment:
Incorrect formatting (ie there's a tab here).

Reply:
Corrected the indentation of the line 1030.